File: scratch/find_cristian.py
import urllib.request
import json

url = "http://localhost:8000/api/get_productores.php"
with urllib.request.urlopen(url) as resp:
    producers = json.loads(resp.read().decode('utf-8'))

for p in producers:
    name = p.get('nombre_completo', '')
    if 'cristian' in name.lower() or 'morales' in name.lower():
        print(f"Found Producer: ID={p['id']}, Name='{p['nombre_completo']}', Document='{p.get('numero_documento')}', Vereda='{p.get('vereda')}'")

File: scratch/find_cristian.php
<?php
require_once __DIR__ . '/../api/db_config.php';

$stmt = $pdo->query("SELECT id, nombre_completo, vereda, numero_documento FROM productores_sumapaz WHERE nombre_completo LIKE '%Cristian%' OR nombre_completo LIKE '%Morales%'");
$rows = $stmt->fetchAll(PDO::FETCH_ASSOC);

echo "Found Cristian records:\n";
print_r($rows);
